analyzers/tests/SonarAnalyzer.UnitTest/TestCases/InheritedCollidingInterfaceMembers.CSharp11.cs

```diff
@@ -12,7 +12,7 @@ public interface IAbstractMethodSecond
         public static abstract string StaticAbstractMethod(string value);
     }
 
-    public interface IAbstractMethodCommon : IAbstractMethodFirst, IAbstractMethodSecond { } // FN
+    public interface IAbstractMethodCommon : IAbstractMethodFirst, IAbstractMethodSecond { } // Compliant: static methods are not inherited
```

Comment:
How did you come to this conclusion? Even though IVirtualMethodCommon does not define a method called StaticVirtualMethod, it is still possible to do something like:
```
void Method<T>() where T : IVirtualMethodCommon
{
    T.StaticVirtualMethod("Hello there");
}
```

Reply:
You are correct @csaba-sagi-sonarsource . 

After a lot of research and internal talks, i re-enabled support for static methods.